.github/workflows/rust.yml

```
@@ -31,8 +31,8 @@ jobs:
       with:
         toolchain: stable
         override: true
-    - name: Cache dependencies
-      uses: Swatinem/rust-cache@v2.2.0
+    # - name: Cache dependencies
```

Maybe we could comment on why we're disabling the cache, to spare future readers from guessing

```suggestion
    # Disable cache due to disk space issues with Windows workers in CI
    # - name: Cache dependencies
```